relatorio-filtro-cursos.html

```diff
@@ -1,28 +1,143 @@
 <html>
 <head>
 <meta charset="UTF-8">
+<title>Relatório Vaas School</title>
 </head>
 <body>
 <table>
 <thead>
 <tr>
-<th>Id Curso</th><th>Nome Curso</th><th>Tempo de Finalização do Curso</th><th>Id Subcategoria</th><th>Nome Subcategoria</th></tr>
+<h2>Categoria Ativa</h2>
+<th>Nome Categoria</th><th>Código Categoria</th><th>Ordem Categoria/th><th>Descrição Categoria</th><th>Status Categoria</th><th>Icone Categoria</th><th>Cor Categoria</th></tr>
 </thead>
 <tbody>
 <tr>
+<td>Programação</td>
+<td>programacao</td>
 <td>1</td>
+<td>Programe nas principais linguagens e plataformas. Iniciantes são bem vindos nos cursos de lógica e JavaScript.</td>
+<td>true</td>
+<td>https://www.alura.com.br/assets/api/formacoes/categorias/512/programacao-transparent.png</td>
+<td>#00c86f</td>
+</tr>
+<tr>
+<td>DevOps</td>
+<td>devops</td>
+<td>2</td>
+<td>Aprenda Git. Entenda a entrega contínua. Estude Linux. Gerencie servidores na nuvem. Explore o mundo de Internet das coisas e da robótica.</td>
+<td>true</td>
+<td>https://www.alura.com.br/assets/api/formacoes/categorias/512/devops-transparent.png</td>
+<td>#f16165</td>
+</tr>
+</tbody>
+</table>
+<table>
+<thead>
+<tr>
+<h2>Subcategoria Ativa</h2>
+<th>Nome Subcategoria</th><th>Código Subcategoria</th><th>Ordem Subcategoria/th><th>Descrição Subcategoria</th><th>Status Subcategoria</th><th>Categoria Subcategoria</th></tr>
```

Comment:
Sobrou esse `/th>` aqui também...

```suggestion
<th>Nome Subcategoria</th><th>Código Subcategoria</th><th>Ordem Subcategoria<th>Descrição Subcategoria</th><th>Status Subcategoria</th><th>Categoria Subcategoria</th></tr>
```